slack-base/src/main/java/com/hubspot/slack/client/models/blocks/UnknownBlock.java

```diff
@@ -0,0 +1,18 @@
+package com.hubspot.slack.client.models.blocks;
+
+import java.util.Optional;
+
+public class UnknownBlock implements Block {
+  protected UnknownBlock() {
+  }
+
+  @Override
+  public String getType() {
+    return "";
```

Comment:
is there any way we can make the unknown block actually pick up whatever type they sent back to us, _or_ `UNKNOWN` if there's nothing? I'd love to be able to inspect this in some cases and actually see what type we're missing.

Reply:
I didn't want to go with UNKNOWN (or any concrete value) incase it ever clashes with one of slack's actual blocks (since they have undocumented ones).

I'll investigate capturing the actual value returned from slack though.

Reply:
Jackson doesn't support capturing the original type value of an unknown value.  I'll need to write customer serialiser and deserialisers for that.

Reply:
I think this is a case where that would be worthwhile, since having to manually debug and figure out the raw strings is going to be pretty painful

Reply:
There's no way to write the deserialiser without having to hand code the deserialisation of every field in every block/block element type.  For now, let's go with `unknown` as the type.  If someone needs support for a new type of block or element, they can always add it.

Reply:
Ok I buy that. Let's do it